### PR TITLE
Add cover URL to Standard Ebooks import job

### LINKED ISSUE #5994: Add covers to standard ebooks script
Subtask of #5792 

We discovered in #5993 (Thank you @hornc !) that our import endpoint already supports covers! We just need `cover: "http://foo..."` in the json.

Now we just need to update the standard ebooks importer to include the cover in the import request.

### Related files

https://github.com/internetarchive/openlibrary/blob/master/scripts/import_standard_ebooks.py

### PR DESCRIPTION
**Pull request by jimchamp**
Closes #5994

Adds full-size cover image URL to Standard Ebooks import items.

Also, does not write last updated date to file on dry-run.

### CODE REVIEW

File: scripts/import_standard_ebooks.py [hunk 1]
```diff
@@ -26,7 +27,11 @@ def get_feed():
 def map_data(entry) -> dict[str, Any]:
     """Maps Standard Ebooks feed entry to an Open Library import object."""
     std_ebooks_id = entry.id.replace('https://standardebooks.org/ebooks/', '')
-    return {
+    image_uris = filter(lambda link: link.rel == IMAGE_REL, entry.links)
+    print (f'image uris: {image_uris}')
+    print (f'type: {type(image_uris)}')
```
Comment (jimchamp) [inline scripts/import_standard_ebooks.py:32]:
```suggestion
```

**Review by cdrini — APPROVED** [review]
Nice! Confirmed job runs successfully, and now includes covers!

```
PYTHONPATH=. ./scripts/import_standard_ebooks.py conf/openlibrary.yml --dry-run
```

**Comment by jimchamp** [comment]
Thanks!  Corresponding OL Client PR is here: https://github.com/internetarchive/openlibrary-client/pull/280

**Comment by cdrini** [comment]
(Side note, I squashed your final commit here, since it doesn't provide too much useful context for anyone who might do "code archeology" on this code)

**Pull request merged**